examples/flight_udtf_llm_example.py

```diff
@@ -31,7 +26,6 @@ def get_client():
 
 
 def extract_sentiment(text):
-    @retry(wait=wait_random_exponential(min=1, max=60), stop=stop_after_attempt(3))
     def completion_with_backoff(**kwargs):
```

Comment:
if we don't have `tenacity`/`retry`, this should no longer be called `completion_with_backoff`